resources/run_adl.sh

```diff
@@ -56,6 +63,8 @@ MONGO_DOWNLOAD_UBUNTU=mongodb-linux-x86_64-ubuntu1804-5.0.4.tgz
 MONGO_DOWNLOAD_REDHAT=mongodb-linux-x86_64-rhel70-5.0.4.tgz
 # macOS
 MONGO_DOWNLOAD_MAC=mongodb-macos-x86_64-5.0.4.tgz
+# Windows
+MONGO_DOWNLOAD_WIN=mongodb-windows-x86_64-5.0.4.zip
```

Comment:
I wonder if/when we want to bump these versions from 5.0 to something later (a rapid release or 6.0). Just a thought; no action needed on this PR.

Reply:
I was thinking the same when I added this line, whether to update these to 5.0.10.   One thought I had was to use curator for the downloads which would at least get the updated rapid release.  
I can create an investigation ticket to track this and find how we want to handle this going forward.